### DISCUSSION TITLE
Is that possible to modify this model to do language translation?

### DISCUSSION THREAD

**Discussion by code-cp** [karpathy/llm.c#499] [General]
I am interested in language translation, but I am not sure whether there are similar small size datasets like tiny stories that is for translation

Also, what kind of modification do we need to make so that the model can do translation?

**Comment by TUrazokAI** [comment]
Hi,

I'm also very interested in seeing an adaptation for translation. It would be fascinating to explore how the model can be modified for this purpose!
